numba/__init__.py

```diff
@@ -21,7 +21,7 @@
 njit = decorators.njit
 
 # Re export vectorize decorators
-from .npyufunc import vectorize, guvectorize
+from .npyufunc import vectorize, guvectorize, parallel
```

Comment:
Is there a reason to re-export the parallel module? (I can't see a reason for it)